client/src/components/Doctor/UpdateProfile.tsx

```diff
@@ -0,0 +1,78 @@
+import axios from 'axios';
+import React, { FunctionComponent, useContext, useEffect, useState } from 'react';
+
+import { AuthContext } from '../../providers/AuthProvider';
+import { useAuthFetch } from '../../hooks/api';
+import { HealthContext } from '../../providers/HealthProvider';
+interface AppointmentDetails {
+  name: string;
+  patient_email: string;
+  appointment_id: string;
+  patient_name: string;
+  created_at: string;
+  appointment_time: string;
+  appointment_status: string;
+}
+
+const UpdateProfile: FunctionComponent<{}> = () => {
+  const [address, setAddress] = useState('');
+  const [hc, setHc] = useState('');
+  const { updateProfile, currentAccount } = useContext(HealthContext);
+  const account = currentAccount || '';
+
+  let providers = [
+    {
+      name: 'Kaiser',
+      address: '0xB74357Df49Ed35Ec1AEc5efdA41f5A5D846fAa8e',
+    },
+    {
+      name: 'Sutter Health Care',
+      address: '0xEE619586e0826CA2F00772701eADc2065E5D5A47',
+    },
+    {
+      name: 'El Camino',
+      address: '0x8eda1014b9177d464306935e8fcf9fd27c20aa08',
+    },
+  ];
```

Comment:
why are we hardcoding the providers here?

Reply:
Using mock data for now to implement the functionality. This will be replaced once integration to get all the health care providers is available on the main.